articles/azure-monitor/insights/container-insights-hybrid-setup.md

```diff
@@ -236,6 +236,9 @@ To first identify the full resource ID of your Log Analytics workspace required
 
 ## Install the chart
 
+>[!NOTE]
```

Comment:
Suggest rewording to - The following commands are applicable only for Helm version 2. Use of the `--name` parameter is not applicable with Helm version 3.

Reply:
Thank you for the feedback , updated the Note accordingly.